sdk/helper/envutil/envutil.go

```diff
@@ -0,0 +1,19 @@
+package envutil
```

Comment:
So I think I gave you bad advice. Because `api/` uses environment variables and `sdk/` imports `api/`, we've gotta move this.